apiserver/facades/controller/caasapplicationprovisioner/provisioner.go

```diff
@@ -56,6 +56,12 @@ type ControllerConfigService interface {
 	WatchControllerConfig() (corewatcher.StringsWatcher, error)
 }
 
+// ModelConfigService provides access to the model configuration.
+type ModelConfigService interface {
```

Comment:
Thank you for adding this.

Can we move this to a new file `services.go`. This follow the convention we already have established for state and when we start bring in more services for the other components it will help keep them tidy and organised.